src/views/Notices/Notices.vue

```diff
@@ -21,5 +21,6 @@ export default {
 .notice {
   overflow-x: hidden;
   white-space: break-spaces;
+  text-align: justify;
```

Comment:
Why is this change required?

Reply:
Looks more clean

Before
<img width="1728" alt="Screenshot 2024-06-25 at 1 14 12 PM" src="https://github.com/ibm-openbmc/webui-vue/assets/69679625/048c2dda-14fa-4fb4-8886-bbf8997f96c8">


After
<img width="1728" alt="Screenshot 2024-06-25 at 1 13 25 PM" src="https://github.com/ibm-openbmc/webui-vue/assets/69679625/8e3f32a7-7af9-4bd4-93a0-bc82abeffaec">

Reply:
Okay